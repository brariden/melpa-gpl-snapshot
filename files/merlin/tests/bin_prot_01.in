#!/bin/bash
. functions.sh || exit 1

reset bin_prot_01.ml
package bin_prot
tell <<END
module Test_binprot = struct 
  type t = T with bin_io 
end
END
cmd dump env
