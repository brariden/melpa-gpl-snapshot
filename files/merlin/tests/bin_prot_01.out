["return",{"cursor":{"line":1,"col":0},"marker":false}]
["return",{"result":true}]
["return",{"cursor":{"line":1,"col":0},"marker":false}]
["return",{"cursor":{"line":4,"col":0},"marker":false}]
["return",{"cursor":{"line":4,"col":0},"marker":false}]
["return",[["File \"bin_prot_01.ml\", line 2, characters 7-24","module Test_binprot :\n  sig\n    type t = T\n    val bin_size_t : t -> int\n    val bin_write_t : t Bin_prot.Write.writer\n    val bin_writer_t : t Bin_prot.Type_class.writer\n    val bin_read_t : t Bin_prot.Read.reader\n    val __bin_read_t__ : (int -> t) Bin_prot.Read.reader\n    val bin_reader_t : t Bin_prot.Type_class.reader\n    val bin_t : t Bin_prot.Type_class.t\n  end"]]]
